packages/indexing/src/tables-manager.ts

```diff
@@ -32,17 +31,7 @@ type ConfigTable = {
   readonly validSchema: object
 }
 
-/**
- * Create a list of db column info for relations in a given model
- */
-function relationsDefinitionsToColumnInfo(relations?: ModelRelationsDefinition): Array<ColumnInfo> {
-  if (!relations) {
-    return []
-  }
-  return Object.keys(relations).map((keyName) => {
-    return { name: addColumnPrefix(keyName), type: ColumnType.STRING }
-  })
-}
+const CUSTOM_COLUMN_PREFIX = 'custom_'
```

Comment:
what do we mean by 'custom' is this related to autogenerated columns?  are they private to us?

Reply:
ok - i would have prefered like _INTERNAL or something but sure :-)